### PR TITLE
Add flag to set GCP logging endpoint.

### PR DESCRIPTION
**Pull request by shellyvilenko**
Add flag to override the GCP logging endpoint.

### CODE REVIEW

File: logger/logger.go [hunk 1]
```diff
@@ -56,10 +59,11 @@ var (
 // EnvVars defines environment variables that can be used to modify the logging
 // behavior.
 var EnvVars = struct {
-	DisableCloudLogging, DebugLog string
+	DisableCloudLogging, DebugLog, GcpLoggingEndpoint string
 }{
```
Comment (manugarg) [inline logger/logger.go:63]: Minor change:
s/GcpLoggingEndpoint/GCPLoggingEndpoint/

File: logger/logger.go [hunk 2]
```diff
@@ -490,7 +503,7 @@ func (l *Logger) Criticalf(format string, args ...interface{}) {
 
 func envVarSet(key string) bool {
 	v, ok := os.LookupEnv(key)
-	if ok && strings.ToUpper(v) != "NO" && strings.ToUpper(v) != "FALSE" {
+	if ok && strings.ToUpper(v) != "NO" && strings.ToUpper(v) != "FALSE"  && strings.ToUpper(v) != "" {
```
Comment (manugarg) [inline logger/logger.go:506]:
```suggestion
	if ok && strings.ToUpper(v) != "NO" && strings.ToUpper(v) != "FALSE"  && v != "" {
```

**Review by manugarg — COMMENTED** [review]
A couple of minor changes.

File: logger/logger.go (hunk 1 above)
Comment (shellyvilenko) [reply logger/logger.go]: thanks ! fixed :)

**Pull request merged**